modules/delete-acs-global-resources.adoc

```diff
@@ -1,39 +1,66 @@
 // Module included in the following assemblies:
 //
 // * installing/uninstall-acs.adoc
+
 :_mod-docs-content-type: PROCEDURE
 [id="delete-acs-global-resources_{context}"]
 = Deleting global resources
 
 [role="_abstract"]
-You can delete the global resources that {product-title} creates, by using the {ocp} or Kubernetes command-line interface.
+You can delete the global resources that {rh-rhacs-first} creates by using the {ocp} or Kubernetes command-line interface (CLI).
```

Comment:
```suggestion
You can delete the global resources that {rh-rhacs-first} creates by using the {oc-first} or Kubernetes command-line interface (CLI).
```

Reply:
For mentions of the OpenShift CLI (`oc`), use the attribute {oc-first}.

Reply:
I guess this reads fine as is and I don't think its mandatory to necessarily put it like this so I won't block merge over it. In the SSG, I see OpenShift CLI is approved. But I think its preferable to use this attribute, so you get the `oc` in parentheses.